### DISCUSSION TITLE
[질문] pr을 보낼 때 생기는 exception

### DISCUSSION THREAD

**Discussion by boogi-woogi** [TEAM-DAME/Dame-Android#51] [2. Q&A]
![image](https://user-images.githubusercontent.com/88770426/201927359-5ff6e42e-81c8-47c5-bb00-9601a829cee8.png)
pr을 보낼때 계속해서 생기는 오류가 있습니다. 
해당 exception을 찾아보니 https://www.inflearn.com/questions/41731 ...
조금 더 생각해보겠습니다!

**Comment by onseok** [comment]
pr을 보낼 때 오류가 생긴다는 것이 깃허브 액션 CI 하는 과정에서 일어난다는 뜻인가요?

**Reply by boogi-woogi** [reply]
넵 맞습니다
